ui-supplemental/partials/header-content.hbs

```diff
@@ -20,7 +20,7 @@
     </div>
     <div id="topbar-nav" class="navbar-menu">
       <div class="navbar-end">
-        <!-- <a class="navbar-item" href="{{site.url}}">Home</a> -->
+        <a class="navbar-item" href="https://brlcad.org">Home</a>
```

Comment:
Do we need to hard code `https://brlcad.org` here? Can `site.url` replace this string here?

Reply:
The only reason to hardcode it was that site.URL is something that reflects the domain of the documentation site, There is a good chance that this has to be assigned to a subdomain like documentation.brlcad.org because we are using reactjs for the main project which means homepage will refer to root of documentation project not the main project

Reply:
I see, thanks.